formstack-baton-requests/cloud-formation.yaml

```diff
@@ -8,8 +8,8 @@ Parameters:
       - PROD
       - CODE
     Default: CODE
-  SarResultsBucket:
-    Description: Bucket where sar results are uploaded to
+  ResultsBucket:
```

Comment:
I THINK it's okay as each one should have its own initiation reference (which will be in the path) but I've just made some changes just in case to add the requestType to the path in S3. It will definitely make it clearer 👍